src/zcl_abapgit_code_inspector.clas.abap

```diff
@@ -183,9 +190,37 @@ CLASS ZCL_ABAPGIT_CODE_INSPECTOR IMPLEMENTATION.
       AND delflag = abap_false
       AND pgmid = 'R3TR'.                               "#EC CI_GENBUFF
 
+    LOOP AT lt_objs INTO ls_obj.
+
+      IF skip_object( ls_obj ) = abap_true.
+        CONTINUE.
+      ENDIF.
+
+      INSERT ls_obj INTO TABLE lt_objs_check.
+
+    ENDLOOP.
+
     ro_set = cl_ci_objectset=>save_from_list(
       p_name    = mv_name
-      p_objects = lt_objs ).
+      p_objects = lt_objs_check ).
+
+  ENDMETHOD.
+
+
+  METHOD skip_object.
+
+    DATA: ls_trdir TYPE trdir.
+
+    IF is_obj-objtype =  'PROG'.
```

Comment:
IF is_obj-objtype <> 'PROG', then variable `rv_skip` is not set, relies on implicit initial value according to the type

Reply:
I set again CASE to set abap_false in WHEN OTHERS.
I thought don't care if `rv_skip` was abap_false or an initial value